galaxy_importer/collection.py

```diff
@@ -126,6 +130,41 @@ def _extract_tar_shell(tarfile_path, extract_dir):
     subprocess.run(args, cwd=cwd, stderr=subprocess.PIPE, check=True)
 
 
+def check_artifact_file(path_prefix, artifact_file):
+    """Check existences of artifact_file on fs and check the chksum matches
+
+    Args:
+        path_prefix (str): Any file path prefix we need to add to file paths in the
+            CollectionArtifactFile artifact_file
+        artifact_file (CollectionArtifactFile): object with info (name, path) of the file
```

Comment:
would it be more clear to say `file object inside the collection artifact with info (name, path) of the file` to deferentiate  CollectionArtifactFile from the collection artifact file tarball?

Reply:
Well, 'file object' implies a python file object to me. ie, `file_object = open('/tmp/somefile', 'r')`

I might rename CollectionArtifactFile -> CollectionArtifactFileInfo to imply it is not a 'file' itself, but info about a file (name, chksum, etc). Or maybe even 'tarfile' object like and CollectionArtifactFileMemberInfo() ?

Reply:
Trying to avoid something like CollectionArtifactManifestFileMemberInfo because the CollectionArtifactFile class is used in CollectionArtifactManifest and CollectionArtifactFilesManifest.

I could ditch CollectionArtifact* entirely, but will likely also be RoleArtifacts soon.

Reply:
maybe the docstring could say `file inside the collection artifact with info (name, path) of the file` ... i just want it to differentiate somewhere that `artifact_file` is not the artifact tarball file

Reply:
@awcrosby Updated to be

``` python
    def _load_file_manifest(self, path_prefix, file_manifest_file):
        """Load CollectionArtifactFileManifest data from file_manifest_file

        Args:
            path_prefix (str): Any file path prefix we need to add to file paths in the
                CollectionArtifactFile artifact_file
            file_manifest_file (CollectionArtifactFile): object with info about the
                FILES.json in the artifact. The info includes the name, type, path,
                and chksum.
      <...>
      """
````

Reply:
@alikins i dont see the update in the files changed

Reply:
well, for check_artifact_file(), the artifact_file kinda isn't in the artifact, in the sense it has been extracted to fs already.
But I'll clarify the doc string.

Reply:
``` diff
-        artifact_file (CollectionArtifactFile): object with info (name, path) of the file
+        artifact_file (CollectionArtifactFile): object with the expected info about
+            the file on the fs that will be checked.
+            This info includes name, type, path, and checksum.
```